WordPress/Classes/Utility/Networking/RequestAuthenticator.swift

```diff
@@ -221,6 +225,46 @@ class RequestAuthenticator: NSObject {
         }
     }
 
+    private func requestForMappedWPCom(url: URL, cookieJar: CookieJar, username: String, authToken: String, siteID: Int, completion: @escaping (URLRequest) -> Void) {
+        func done() {
+            guard
+                let host = url.host,
+                !host.contains("wordpress.com")
+            else {
+                // The requested URL is to the unmapped version of the domain,
+                // so skip proxying the request through r-login.
+                let request = URLRequest(url: url)
+                completion(request)
+                return
+            }
+
+            let rlogin = "https://r-login.wordpress.com/remote-login.php?action=auth"
+            var components = URLComponents(string: rlogin)
+            var queryItems = components?.queryItems ?? []
+            queryItems.append(URLQueryItem(name: "host", value: host))
```

Comment:
This could be easier to read:

```swift
            queryItems.append(contentsOf: [
                URLQueryItem(name: "host", value: host),
                URLQueryItem(name: "id", value: String(siteID)),
                URLQueryItem(name: "back", value: url.absoluteString)
            ])
```

Feel free to disregard this comment, as it's purely cosmetic and honestly subjective

Reply:
I like it!